src/main/java/seedu/address/logic/Messages.java

```diff
@@ -14,6 +14,7 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_STARTDATE = "Start Date must be earlier than End Date";
```

Comment:
Should the name of the string be INVALID_DATES since, it could be that the end date is the one that is invalid